### DISCUSSION TITLE
Share kitty.conf and custom tab_bar.py

### DISCUSSION THREAD

**Discussion by dimao** [kovidgoyal/kitty#4616] [Q&A] [answered]
Hi, can you share your kitty.conf and custom tab_bar.py script from this video: [Watch kitty in action!](https://download.calibre-ebook.com/videos/kitty.mp4) ?
Thanks!

**Comment by kovidgoyal** [comment, marked as answer]
There is no custom tab bar, the bar you see at the top is a X11 panel using the kitty panel kitten with a program to draw the data that I have written for myself that is not suitable for public release. As for kitty.conf there is almost nothing in it, just the colorscheme (Dimmed Monokai) which you can get by running the kitty themes kitten, and the font, Operator Mono.
